credentialz/credentialz.proto

```diff
@@ -620,14 +645,22 @@ message CaPublicKeyRequest {
 message CaPublicKeyResponse {
 }
 
+// A request to generate a public/private key pair based on a given key
+// specification and return the public key back which will be used to create
+// a hsot SSH certificate.
+message GenerateKeysRequest {
+  KeyGen key_params = 1;
+}
+
+// Response from the target which has the generated public key.
+message GenerateKeysResponse {
```

Comment:
You have to add this to the `RotateHostCredentialsResponse` message at line 584. Otherwise you will not be able to send it over the stream.

Reply:
Done